### PR TITLE
Update IT trans.

### PR DESCRIPTION
**Pull request by dzpex**
Sorry for my delay in file delivery but i had problem with my pc!

### CODE REVIEW

File: src/res/translation/translation_it_IT.ts [hunk 1]
```diff
@@ -938,12 +938,12 @@
...
         <location filename="../../clientdlg.cpp" line="322"/>
         <source>Use &amp;Two Rows Mixer Panel</source>
-        <translation type="unfinished"></translation>
+        <translation>Abilita &amp;Vista su due Righe</translation>
     </message>
     <message>
         <location filename="../../clientdlg.cpp" line="330"/>
         <source>&amp;Clear All Stored Solo and Mute Settings</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```
Comment (trebmuh) [inline src/res/translation/translation_it_IT.ts:946]: It looks like there is a missing translation here. Or is it done on purpose maybe?

**Comment by dzpex** [comment]
ok now its ok!

**Comment by trebmuh** [comment]
@dzpex it looks that you provided the wrong file.
You uploaded the compiled (_translation_it_IT.**qm**_) file instead of the source (_translation_it_IT.**ts**_) file.

**Comment by dzpex** [comment]
Try now....i resend the file from my pc

**Comment by trebmuh** [comment]
That's good now for the .ts file.
I'm not sure the qm file is useful. @corrados will probably tells you.

Enjoy the end of your sunday!

**Comment by corrados** [comment]
> I'm not sure the qm file is useful

It does not hurt but usually I re-generate this file anyway on my PC.

**Pull request merged**